leap-year.clj

```diff
@@ -0,0 +1,8 @@
+(ns exercises.leap-year)
+
+(defn leap-year?
+  [year]
+  (cond (zero? (mod year 400)) true
```

Comment:
💅 I'm a big fan of how `(->)` reads. Maybe try it out on this scenario to see how you feel about it